src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -1,27 +1,34 @@
 import { createSelector } from '@reduxjs/toolkit'
```

Comment:
All the selectors using `stakingData` can probably go away.
Similarly, there's a lot of duplication across selectors, it should be cleaned up before reopening the PR.